.github/workflows/main.yml

```diff
@@ -0,0 +1,58 @@
+# This is a basic workflow to help you get started with Actions
+
```

Comment:
create a proper folder structure that makes it easy to add dependencies for postgres and others.